test/index.test.js

```diff
@@ -1,62 +1,74 @@
-  describe.skip('#compress', () => {
-    const buffer = Buffer.from('test');
+  test('compress() returns a Nodejs buffer', async function () {
+    expect(await compress(Buffer.from([1, 2, 3]))).to.be.instanceOf(Buffer);
+  });
 
-    context('when not providing a compression level', () => {
-      it('returns a compressed buffer', async () => {
-        const result = await compress(buffer);
-        expect(await decompress(result)).to.deep.equal(buffer);
-      });
+  test('decompress() with empty buffer', async function () {
+    expect(await decompress(Buffer.from([]))).to.deep.equal(Buffer.from([]))
+  })
+});
+
+/**
+ * @param {import('../index').decompress} decompress
+ * @param {import('../index').compress} compress
+ */
+function testSuite(decompress, compress) {
+  return function () {
+    test('empty', async function () {
+      const input = Buffer.from('', 'utf8');
+      const result = await decompress(await compress(input));
+      expect(result.toString('utf8')).to.deep.equal('');
     });
 
-    context('when providing a compression level', () => {
-      context('when the level is valid', () => {
-        it('returns a compressed buffer', async () => {
-          const result = await compress(buffer, 1);
-          expect(await decompress(result)).to.deep.equal(buffer);
-        });
-      });
-
-      context('when the level is invalid', () => {
-        context('when the level is too high', () => {
-          it('returns a compressed buffer', async () => {
-            const result = await compress(buffer, 100);
-            expect(await decompress(result)).to.deep.equal(buffer);
-          });
-        });
-
-        context('when the level is too low', () => {
-          it('returns a compressed buffer', async () => {
-            const result = await compress(buffer, -100);
-            expect(await decompress(result)).to.deep.equal(buffer);
-          });
-        });
-      });
+    test('one element', async function () {
+      const input = Buffer.from('a', 'utf8');
+      const result = Buffer.from(await decompress(await compress(input)));
+      expect(result.toString('utf8')).to.deep.equal('a');
     });
-  });
 
-  describe.skip('#decompress', () => {
-    context('when decompressing invalid data', () => {
-      it('includes a stack trace', async () => {
-        try {
-          await decompress(Buffer.from('invalid'));
-        } catch (error) {
-          expect(error.message).to.equal('zstd: Unknown frame descriptor');
-          expect(error.stack).to.match(/at decompress/);
-        }
-      });
+    test('typical length string', async function () {
+      const input = Buffer.from('hello, world! my name is bailey', 'utf8');
+      const result = Buffer.from(await decompress(await compress(input)));
+      expect(result.toString('utf8')).to.deep.equal('hello, world! my name is bailey');
     });
-  });
-});
+
+    test('huge array', async function () {
```

Comment:
```suggestion
    it('huge array', async function () {
```

Why are we switching to tdd for this repo? we use bdd everywhere else

Reply:
I don't feel strongly but I wouldn't introduce the inconsistency here